### PR TITLE
BMC: added method to get system's power state.

### PR DESCRIPTION
**Pull request by greyerof**
bmc.SystemPowerState() returns the PowerState (string) field from the bmc's configured ComputerSystem.

### CODE REVIEW

File: pkg/bmc/bmc.go [hunk 1]
```diff
@@ -472,6 +472,40 @@ func (bmc *BMC) SystemPowerCycle() error {
...
+func (bmc *BMC) SystemPowerState() (string, error) {
+	if valid, err := bmc.validateRedfish(); !valid {
+		return "", err
+	}
+
+	glog.V(100).Info("Collecting current power state from bmc's redfish endpoint")
+
+	redfishClient, cancel, err := redfishConnect(
+		bmc.host,
+		bmc.redfishUser.Name, bmc.redfishUser.Password,
```
Comment (kononovn) [inline pkg/bmc/bmc.go:486]: Please move bmc.redfishUser.Password, to the next line according to https://github.com/openshift-kni/eco-goinfra#functions-format

**Review by kononovn — CHANGES_REQUESTED** [review]
Just 1 cosmetic comment.

File: pkg/bmc/bmc.go (hunk 1 above)
Comment (greyerof) [reply pkg/bmc/bmc.go]: Done in https://github.com/openshift-kni/eco-goinfra/pull/442/commits/7c150595b63e7b41722bfc565b1898a00020db4b

**Review by kononovn — APPROVED** [review]
/lgtm

**Review by klaskosk — APPROVED** [review]
lgtm

**Pull request merged**